components/org.wso2.carbon.identity.provisioning.connector.google/src/main/java/org/wso2/carbon/identity/provisioning/connector/google/GoogleProvisioningConnector.java

```diff
@@ -186,23 +182,22 @@ protected void updateUser(ProvisioningEntity provisioningEntity)
             }
         } else {
             throw new IdentityProvisioningException(
-                    "Cannot updating Google user, provisionedIdentifier is invalide.");
+                    "Cannot updating Google user, provisionedIdentifier is invalid.");
         }
 
-        if (log.isTraceEnabled()) {
-            log.trace("Ending updatingUser() of " + GoogleProvisioningConnector.class);
+        if (log.isDebugEnabled()) {
+            log.debug("Ending updatingUser() of " + GoogleProvisioningConnector.class);
```

Comment:
How about logging some identifier related to the updated user here?

Reply:
This log is unnecessary it seems. Will remove it